lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -74,15 +75,6 @@ def do_not_generate_strong_types?(constant)
             !(constant.singleton_class < Object.const_get(:StrongTypeGeneration))
         end
 
-        sig { params(type: String).returns(String) }
-        def as_nilable_type(type)
-          if type.start_with?("T.nilable(") || type == "T.untyped"
```

Comment:
Looks like a copy/paste of `as_nilable_type` from the base compiler class but was (presumably unintentionally?) not updated to handle `::T.nilable(` or `::T.untyped` like the other was.

Reply:
Yeah, good catch!